.github/workflows/sycl-windows-build.yml

```diff
@@ -164,6 +165,19 @@ jobs:
       if: always() && !cancelled() && contains(inputs.changes, 'libdevice')
       run: |
         cmake --build build --target check-libdevice
+    - name: Generate/diff new ABI symbols
+      if: always() && !cancelled() && contains(inputs.changes, 'sycl')
+      shell: bash
+      run: |
+        LLVM_BIN_PATH="build/bin" python.exe src/sycl/tools/abi_check.py --mode dump_symbols --output build/new_sycl_symbols_windows.dump build/bin/sycl?.dll
+        diff -Naur src/sycl/test/abi/sycl_symbols_windows.dump build/new_sycl_symbols_windows.dump || true
+    - name: Upload new ABI symbols
```

Comment:
What's the motivation behind diff'ing and uploading the ABI symbols for Windows?

Reply:
Have you never wished it was there? I personally hate searching for a local windows machine to regenerate those...

Reply:
Not really. We have a LIT test to check if ABI has changed or not, so I didn't had to manually regenerate and diff the ABI symbol dumps.
How do you plan to use the uploaded ABI symbol dump? ABI-breaks aren't frequent so, I'm not entirely convinced if this should be a part of the CI. Also, why the diff'ing?

Reply:
Discussed offline. One potential use case for symbol dump could be when new ABI symbols are added, which is not so infrequent.